packages/shopping-cart/src/types.ts

```diff
@@ -507,6 +507,8 @@ export interface ResponseCartProduct {
 
 	price_tier_minimum_units?: number | null;
 	price_tier_maximum_units?: number | null;
+	price_tier_transform_quantity_divide_by?: number | null;
```

Comment:
Could we put comments above each of these properties to explain what they mean?

Reply:
Good call, I've added comments to the added properties. I've mostly just copied existing ones from backend.

Reply:
Lovely! 👌 Thank you!